GetF!t/Modules/Exercises/Success/SuccessView.swift

```diff
@@ -31,10 +31,10 @@ struct SuccessView: View {
             }
 
             VStack {
-                HeaderView(titleText: LocalizedStringProvider.SuccesPage.success)
+                HeaderView()
                 Spacer()
                 RaisedButton(buttonText: LocalizedStringProvider.Button.continuE) {
```

Comment:
stále to tu je z veľkým E. V LocalizedStringProvider si máš všade `` .. má zmysel to dávať k názvu premennej iba keď chceš použiť kľúčové slovo. Prosím odmaž to tam od všetkých vecí okrem continue

Reply:
done